runtime/jvmti/jvmtiHook.c

```diff
@@ -3631,3 +3643,21 @@ jvmtiHookVmDumpEnd(J9HookInterface** hook, UDATA eventNum, void* eventData, void
 
 	TRACE_JVMTI_EVENT_RETURN(jvmtiHookVmDumpEnd);
 }
+
+/* Check if a JVMTI event should be dispatched by checking the flag set by
+ * notifyJvmtiHideFrames() and the presence of the JvmtiMountTransition
+ * annotation.
+ *
+ * Note that the method should also be check for Method Entry and Method Exit event
+ */
+static BOOLEAN
+shouldPostEvent(J9VMThread *currentThread, J9Method *method)
```

Comment:
ifdef to return TRUE for `JAVA_SPEC_VERSION < 20`? compilation will fail since `CFR_METHOD_EXT_JVMTIMOUNTTRANSITION_ANNOTATION` is only defined in `JAVA_SPEC_VERSION >= 20`.